resources/views/users/includes/membershipform_include.blade.php

```diff
@@ -1,49 +1,33 @@
-<h2>Becoming a member of Study Association Proto</h2>
-
 <p>
     The undersigned ...
 </p>
-
 <p>
-    <strong>{{ $user->name }}</strong>, born on
-    <strong>{{ date('F j, Y', strtotime($user->birthdate)) }}</strong>
+    @php($date = new DateTime($user->birthdate))
+    <b>{{ $user->name }}</b>, born on <b>{{ $date->format('M, d, Y') }}</b>,
 </p>
-
 <p>
     ... wishes to become a member of Study Association Proto.
 </p>
-
 <p>
-    The undersigned is aware of the Bylaws (NL: Statuten) and Rules & Regulations (NL: Huishoudelijk Regelement) of the association and promises to follow them.
-    More information on the before mentioned documents can be found on the <a href="https://wiki.proto.utwente.nl/proto/start" target="_blank">DokuWiki</a>.
+    The undersigned is aware of the
+    <a href="https://wiki.proto.utwente.nl/_media/proto/statues_v3_eng_censor.pdf" target="_blank">Bylaws</a>
+    (NL: <a href="https://wiki.proto.utwente.nl/_media/proto/statuten_v3_nl_censor.pdf" target="_blank">Statuten)</a> and
+    <a href="https://wiki.proto.utwente.nl/_media/proto/rules_and_regulations_s.a._proto_en_.pdf" target="_blank">Rules & Regulations</a>
+    (NL: <a href="https://wiki.proto.utwente.nl/_media/proto/rules_and_regulations_s.a._proto_nl_.pdf" target="_blank">Huishoudelijk Regelement)</a>
+    of the association and promises to follow them.
```

Comment:
The Dutch versions of these documents are leading, so only referencing those in parentheses seems a bit odd

Reply:
But at the same time the contract is in English, so you have to be clear that the documents you are referring to are in a different language.

Reply:
I would maybe just add a note that the dutch versions are leading